doc/Building.md

```diff
@@ -7,6 +7,8 @@ run it with `--appimage-extract`.
 
 You can skip most of the following instructions if desired, and use our premade Docker image instead. In that case the only requirements are Git and Docker. See [the virtual development environment README](https://github.com/koreader/virdevenv) for more information.
 
+**Note:** If you want to use WSL then you need to export a sane PATH first because windows append their own directories to it. You'll also need to [install a XServer](https://virtualizationreview.com/articles/2017/02/08/graphical-programs-on-windows-subsystem-on-linux.aspx). If you need more info please read https://github.com/koreader/koreader/issues/6354
```

Comment:
```suggestion
**Note:** If you want to use WSL then you'll need to export a sane PATH first, because Windows appends its own directories to it. You'll also need to [install an XServer](https://virtualizationreview.com/articles/2017/02/08/graphical-programs-on-windows-subsystem-on-linux.aspx). If you need more info please read <https://github.com/koreader/koreader/issues/6354>.
```